tgui/packages/tgui/interfaces/ChemFilter.tsx

```diff
@@ -11,37 +11,30 @@ type Data = {
 type Props = {
   title: string;
   list: string[];
-  reagentName: string;
-  onReagentInput: (str: string) => void;
+  btitle: string;
+  bcolor: string;
 };
 
 export const ChemFilterPane = (props: Props, context) => {
   const { act } = useBackend(context);
-  const { title, list, reagentName, onReagentInput } = props;
+  const { title, list, btitle, bcolor } = props;
```

Comment:
Do not abbreviate names. Fully expand this to `buttonTitle` and `buttonColor`.

Reply:
My bad ill fix that